src/components/NumberInput/NumberInput.unit.spec.jsx

```diff
@@ -167,12 +167,70 @@ describe('<NumberInput>', () => {
     tree.unmount();
   });
 
+  it('has aria-describedby attribute when ariaDescribedby is set', () => {
+    const tree = shallow(
+      <NumberInput
+        field={testValue}
+        label="my label"
+        ariaDescribedby="some-id"
+        onValueChange={() => {}}
+      />,
+    );
+
+    // Ensure no error.
+    expect(tree.find('.usa-input-error')).to.have.lengthOf(0);
+
+    // No error means no aria-describedby to not confuse screen readers.
+    const inputs = tree.find('input');
+    expect(inputs).to.have.lengthOf(1);
+    expect(inputs.prop('aria-describedby')).to.equal('some-id');
```

Comment:
The comments here are confusing to me. We don't want an error, but that means that there shouldn't be an `aria-describedby`, yet we still expect to find it?

Reply:
Yeah, that comment is bad... what I'm trying to say is that the `aria-describedby` shouldn't be added as an attribute if there are no errors or no `ariaDescribedby` prop. In this case, there is no error but there is an `ariaDescribedby` prop, so it still adds the attribute with _only_ the id passed into it. I'll fix the comment.